customer-service/src/main/java/com/spaghettininjas/yaposs/repository/entity/Customer.java

```diff
@@ -0,0 +1,50 @@
+package com.spaghettininjas.yaposs.repository.entity;
+
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "customer")
+public class Customer {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String email;
+
+    private  String loyaltyLevel;
+
+    private String loyaltyPoints;
+
+    private String name;
+
+    public Customer(String email, String loyaltyLevel, String loyaltyPoints, String name) {
```

Comment:
Yet still manually wrote the constructor.

[`@NoArgsConstructor`, `@RequiredArgsConstructor` or `@AllArgsConstructor`](https://projectlombok.org/features/constructor) could be used

Reply:
If you use @AllArgsConstructor, then the ID will get there, and hibernate does not work without the constructor

Reply:
https://spring.io/guides/gs/accessing-data-jpa/